<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qDatainfo</class>
 <widget class="QDialog" name="qDatainfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qDatainfo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">border-style: solid;
border-width: 0.5px;
border-color: #A0A0A0;</string>
           </property>
           <property name="text">
            <string>Sample Rate</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="cb_rate">
           <item>
            <property name="text">
             <string>8192</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>44100</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>88200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>176400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>400000</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>441000</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">border-style: solid;
border-width: 0.5px;
border-color: #A0A0A0;</string>
           </property>
           <property name="text">
            <string>Precision</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="cb_precision">
           <item>
            <property name="text">
             <string>8bit</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16bit</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24bit</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>32bit</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>64bit</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">border-style: solid;
border-width: 0.5px;
border-color: #A0A0A0;</string>
           </property>
           <property name="text">
            <string>Point</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="cb_point">
           <item>
            <property name="text">
             <string>32</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>64</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>128</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>256</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>512</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1024</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2048</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4096</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">border-style: solid;
border-width: 0.5px;
border-color: #A0A0A0;</string>
           </property>
           <property name="text">
            <string>Sample Length</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="line_length"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pb_continue">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
